docs/source/cli_help/anno.rst

```diff
@@ -22,12 +22,7 @@ then clears the annotation for the current branch (or a branch specified with ``
 If invoked with ``-H`` or ``--sync-github-prs``, annotates the branches based on their corresponding GitHub PR numbers and authors.
 Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
 
-To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
-a GitHub API token with ``repo`` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
-
-    1. ``GITHUB_TOKEN`` env var,
-    2. current auth token from the ``gh`` GitHub CLI,
-    3. current auth token from the ``hub`` GitHub CLI.
+.. include:: github_api_access.rst
```

Comment:
Nit (optional, can be skipped if can't be done smoothly):

![image](https://user-images.githubusercontent.com/3383210/156377841-6790c792-f0c8-4bf7-83f6-dc7283fc2e2c.png)

To keep the narrative more fluid, I'd wrap the part about GitHub auth into some kind of tooltip/note/frame... I'm pretty sure rst should have a support for sth like that

Reply:
Like e.g. here https://tox.wiki/en/latest/config.html#conf-install_command